solr/solr-ref-guide/modules/query-guide/pages/faceting.adoc

```diff
@@ -168,7 +166,7 @@ The `facet.offset` parameter indicates an offset into the list of constraints to
 |Optional |Default: `0`
 |===
 +
-The `facet.mincount` parameter specifies the minimum counts required for a facet field to be included in the response.
+The minimum count required for a facet field to be included in the response.
 If a field's counts are below the minimum, the field's facet is not returned.
```

Comment:
Wondering about "field" versus "term" wording here. WDYT?

```suggestion
The minimum count required for a facet field term to be included in the response.
If a term's count is below the minimum, the term is not returned.
```

Reply:
Good question! I think I might leave this as is, since I don't fully understand how these terms are supposed to differ. These terms really need a good untangling in this document. If I get the time I'll tackle this in a separate PR to tidy it up!